frontend/src/mixins/shootItem.js

```diff
@@ -129,7 +130,7 @@ export const shootItem = {
       return this.shootSpec.region
     },
     shootZones () {
-      return uniq(flatMap(get(this.shootSpec, 'provider.workers'), 'zones'))
+      return compact(uniq(flatMap(get(this.shootSpec, 'provider.workers'), 'zones')))
```

Comment:
was this a bugfix for a different issue? If yes, it would be better to fix such things in a separate PR

Reply:
Yes this is a bug. For non zoned clusters `shootZones` returned [undefined] which resulted in wrong UI rendering where we check for `zones.length`
I know ... a separate PR would be (even) better but it's a one-line fix and related to this PR so I think we can keep it...